src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -1709,7 +1729,12 @@ class AddItemsPage extends Component {
                 </button>
                 <button
                   type="submit"
-                  onClick={() => this.submitRequest(values.lineItems)}
+                  onClick={() => inactiveProductValidation({
+                      lineItems: values.lineItems,
+                      callback: this.submitRequest,
+                      valuesForCallback: values.lineItems,
```

Comment:
passing a `valuesForCallback` argument feels unnecessary since you are not doing anything with this value but just passing it to `callback`.
I sggest to remove `valuesForCallback` and just pass a callback function like
```js
  onClick={() => inactiveProductValidation({
                      lineItems: values.lineItems,
                      callback: () => this.submitRequest(values.lineItems),
                })
```